flink/values.yaml

```diff
@@ -287,3 +287,22 @@ zookeeper:
       memory: 1256Mi
   persistence:
     enabled: true
+
+#secret:
+#  secrets:
+#    - name: kerberos
+#      mountPath: /kerberos
+#    - name: kafka-ssl
+#      mountPath: /etc/certs
+#  sealedSecrets:
+#    # The encrypted raw file sealed secrets generated for example with
+#    # kubeseal --raw --from-file=... --controller-name sealed-secrets --scope namespace-wide
+#    sealedSecretFiles:
+#      file1: ...
+#      file2: ...
+#    sealedSecretFilesPath: /etc/sealed
+#    # The encrypted raw env sealed secrets generated for example with
+#    # echo -n password | kubeseal --raw --from-file=/dev/stdin --controller-name sealed-secrets --scope namespace-wide
+#    sealedSecretEnvs:
```

Comment:
>> helm template .
Error: template: flink/templates/taskmanager.yaml:76:26: executing "flink/templates/taskmanager.yaml" at <.Values.secret.sealedSecrets.sealedSecretEnvs>: nil pointer evaluating interface {}.sealedSecrets

Reply:
mb change the first level key 'secret' to something more meaningful like 'bitnamiSealedSecrets', uncomment it and add 'enabled' property ?
```
bitnamiSealedSecrets:
  enabled: false
  secrets: ...
  sealedSecrets:...
  ...
```

Then you can convert all checks to something like:
```
{{- if and .Values.secret.enabled .Values.secret.sealedSecrets.sealedSecretFilesPath  }}
```
it should be processed correctly in case of non existence of  `.Values.secret.sealedSecrets.sealedSecretFilesPath`

Reply:
There is support for simple predefined k8s secrets and the bitnami ones. They work in two different styles as the latter allow you to keep "data" inside your code.

I tried to highlight it in the update.

Reply:
And agree with your pieces of advice. Uncommented defaults should be working now.